src/backend/opencl/magma/laset.cpp

```diff
@@ -7,6 +7,50 @@
  * http://arrayfire.com/licenses/BSD-3-Clause
  ********************************************************/
 
+/***********************************************************************
+ * Based on MAGMA library http://icl.cs.utk.edu/magma/
+ * Below is the original copyright.
```

Comment:
It looks like BSD 3-clause. So we can give the copyright, but the detailed text can be pointed to the existing BSD license file